packaging/darwin/Distribution

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<installer-script minSpecVersion="1.000000">
+    <title>CodeReady Containers</title>
+    <background mime-type="image/png" file="banner.png" scaling="proportional"/>
```

Comment:
need to also add for the banner to appear in systems using dark-mode, ref: https://developer.apple.com/forums/thread/108056
```xml
<background-darkAqua mime-type="image/png" file="banner.png" scaling="proportional"/>
```